src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,166 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove all redundant empty lines, be careful :)